loadable_apps/loadable.mk

```diff
@@ -79,6 +79,7 @@ ifeq ($(CONFIG_ELF_EXCLUDE_SYMBOLS),y)
 	$(Q) $(OBJCOPY) --remove-section .comment $(USER_BIN_DIR)/$(BIN)
 	$(Q) $(STRIP) -g $(USER_BIN_DIR)/$(BIN) -o $(USER_BIN_DIR)/$(BIN)
 endif
+	$(Q) cp $(USER_BIN_DIR)/$(BIN) $(USER_BIN_DIR)/$(BIN).uncomp
```

Comment:
How about adding condition for `CONFIG_SUPPORT_COMMON_BINARY`?

Reply:
It's not the content of COMMON_BINARY. It's for COMPRESSED_BINARY. Let me move to work only with it.

Reply:
Oh sorry. Yes, I mean for compress binary.

Reply:
I applied this to work with COMPRESSED_BINARY enabled.